server/transport_test.go

```diff
@@ -142,8 +211,10 @@ func newServer(t *testing.T, opts ...scepserver.ServiceOption) (*httptest.Server
 	}
 	logger := kitlog.NewNopLogger()
 	e := scepserver.MakeServerEndpoints(svc)
-	handler := scepserver.MakeHTTPHandler(e, svc, logger)
-	server := httptest.NewServer(handler)
+	scepHandler := scepserver.MakeHTTPHandler(e, svc, logger)
+	r := mux.NewRouter()
+	r.Handle("/scep", scepHandler)
+	server := httptest.NewServer(r)
 	teardown := func() {
```

Comment:
Nit, but if it doesn't fan out to too many changes, we can use `t.Cleanup(func() {...})` instead of returning the teardown function, so the caller doesn't have to remember to `defer` that function call.